src/XPurchase/VirtoCommerce.XPurchase/Services/CartAvailMethodsService.cs

```diff
@@ -84,8 +93,14 @@ public async Task<IEnumerable<ShippingRate>> GetAvailableShippingRatesAsync(Cart
             {
                 var taxEvalContext = _mapper.Map<TaxEvaluationContext>(cartAggr);
                 taxEvalContext.Lines.Clear();
-                taxEvalContext.Lines.AddRange(availableShippingRates.SelectMany(x => _mapper.Map<IEnumerable<TaxLine>>(x)));
-                var taxRates = taxProvider.CalculateRates(taxEvalContext);
+                taxEvalContext.Lines.AddRange(availableShippingRates.SelectMany(x => _mapper.Map<IEnumerable<TaxLine>>(x)));
+
+                var taxRates = new List<TaxRate>();

```

Comment:
User cartAggr.Store to get stroe settings
if TaxCalculationEnabled == false there's not need to even get `taxProvider` and `ApplyTaxRates`